tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -4216,4 +4216,82 @@ public function testUpdateWithExistingRelationships(array $data): void
 
         $this->assertEquals(200, $update['headers']['status-code']);
     }
+
+    /**
+     * @depends testCreateDatabase
+     */
+    public function testDeleteRelationAttributeIfRelatedCollectionDeleted(array $data): void
+    {
+        $databaseId = $data['databaseId'];
+
+        // Create level 1 collection
+        $level1 = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'collectionId' => ID::unique(),
+            'name' => 'Level 1',
+            'documentSecurity' => true,
+            'permissions' => [
+                Permission::create(Role::user($this->getUser()['$id'])),
+                Permission::read(Role::user($this->getUser()['$id'])),
+            ],
+        ]);
+        // Create level 2 collection
```

Comment:
let's add a line break here and remove this comment 
```suggestion

```

Reply:
Done